common.js

```diff
@@ -1708,6 +1708,176 @@ const allSoloCars = {
       'all': ['ssp'],
     },
   },
+  'Jaguar': {
+    '120': {
+      'all': ['hs'],
+    },
+    '140': {
+      'all': ['hs'],
+    },
+    '150': {
+      'all': ['hs'],
+    },
+    'F-Type (NOC non-Project 7)': {
```

Comment:
This should be combined with the duplicate model below. NOC stands for Not otherwise classified. This seems like it should be a single model definition `F-Type (non-Project 7)` and `as` and `ssr` should be the possible classes.